lib/restbase.js

```diff
@@ -25,7 +25,7 @@ function RESTBase(options, req) {
         this._parent = par;
         // Remember the request that led to this child instance at each level, so
         // that we can provide nice error reporting and tracing.
-        this._req = par._req;
+        this._req = req;
```

Comment:
I think the intention here is to have a reference to the root request, which this won't preserve.